doc/user/administrator/index.md

```diff
@@ -32,6 +32,19 @@ The panel has a top navbar with a "Return" button to return to the main applicat
 
 ![Admin panel.](images/admin-panel.png)
 
+Viewing Statistics
+==================
+
+When the Administrative Panel is accessed you will be taken to the landing page which is the Statistics Page.
+
+![Admin panel statistics link.](images/admin-panel-stats.png)
+
+This page displays usage statistics for the number of analyses ran, projects created, samples created, users created, and users logged in within the default time period which is set to 7 days.
+
+Clicking on any of the graphs will take you to another page for you to see advanced stats where you can select different time periods as well as, different chart types.
```

Comment:
This is awkwardly worded.  Try `Clicking on a graph will take you to the full details of the graph.  There you can set different time periods to explore the data and select different chart types.`  or something like that.

Reply:
Fixed in 528b7e7b961704290515e51f21b4ff2892ae2052. Thanks